master/internal/rm/agentrm/agent.go

```diff
@@ -141,6 +142,21 @@ func (a *agent) receive(ctx *actor.Context, msg interface{}) error {
 			a.address = msg.Ctx.Request().RemoteAddr[0:lastColonIndex]
 		}
 
+		// When the dev cluster is started with "tools/slurmcluster.sh", an SSH
+		// tunnel is created between the local host and the HPC cluster. Due to
+		// the way the reverse tunnel is created, all the nodes will report a
+		// loopback address of "[::1]". This will cause distributed experiments
+		// to fail. To work around this, use the agent ID, which will be the
+		// node name. As long as the cluster has been configured to resolve the
+		// node names to their respective IP addresses via "/etc/hosts", DNS,
+		// or some other mechanism, this will work.
+		if addr := net.ParseIP(strings.Trim(a.address, "[]")); addr != nil && addr.IsLoopback() {
```

Comment:
can we do something more explicit? like have the agent pass `?local_addr=hostname` as a param when configured to do so or fallback to the SLURM-style rendezvous based on a configuration?

Reply:
or, did you look into anything else?

Reply:
Looking to make an adjustment.  Will post updated review shortly.